Monika After Story/game/script-topics.rpy

```diff
@@ -15751,3 +15751,39 @@ label monika_literature_value:
     m 1eku "Honestly, I think if more people valued books and poems a little more, the world would be a much better place."
     m 1hksdlb "That's just my opinion as president of a literature club, though. {w=0.2}I guess most people wouldn't think that deeply about it."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_pianolessons",
+            category=['music'],
+            prompt="Would you give me piano lessons?",
+            pool=True
+        )
+    )
+
+label monika_pianolessons:
+    if persistent._mas_pm_plays_instrument = True
+        m 1wud "You want piano lessons from me?"
+        m 1rksdlb "But, [player]..."
+        m 3hksdlb "It's not unlikely that {i}you{/i} could be the one teaching {i}me,{/i} ahaha!"
+    else:
+        m 1eka "Um...well...maybe?"
+        m 1eksdla "I'm flattered that you'd ask, but..."
+
+    if persistent.monika_kill:
+        m 3eka "Remember? I told you when I sang Your Reality that I wasn't really good at piano. Like, at all."
```

Comment:
m 3eka "Remember? I told you when I sang Your Reality that I wasn't really good at piano.{w=0.3}{nw}"
        extend 3rkb " Like, at all."

I think this comment could do with a change of expression.